### ISSUE TITLE
Binlister file is assigning bins to wrong events

### ISSUE THREAD

**Issue by sujatasinha** [lucklab/erplab#134]
### Description

[*Description of the bug or feature*]
I have events 2,3 and 4 as the three conditions. 5,6 and 7 are the correct responses, respectively. I want to assign Bin 1 as event 2 followed by event 5. But I see that there are some 2's followed by another event 8 assigned into bin 1. Why is this so? 
<br>
#### Steps to Reproduce

1. [*First Step*]
2. [*Second Step*]
3. [*and so on...*]

<br>
#### Expected behavior:
[*What you expected to happen*]

<br>
#### Actual behavior:
[*What actually happened*]

<br>
#### Versions

|   |   |
|----| ---- | 
| OS version     | [*fill in*] |
| Matlab version | [*fill in*] |
| EEGLAB version | [*fill in*] |
| ERPLAB version | [*fill in*] |

**Comment by sujatasinha** [comment]
I am attaching my event list and the bin lister file here.
[Mental_correct_H1.txt](https://github.com/lucklab/erplab/files/4808954/Mental_correct_H1.txt)
[EVENTLIST-MENTAL.txt](https://github.com/lucklab/erplab/files/4808955/EVENTLIST-MENTAL.txt)

**Comment by sujatasinha** [comment]
What I realize is that: because I have already assigned bins in the event list file "EVENTLIST-MENTAL.txt"(event 2 as bin1; event 3 as bin2; and event 4 as bin 3), when I did the binlister step, those event-2's retained the bin 1 even though it was followed by 8 (incorrect event) instead of event-5.

So, I tried removing bins from the eventlist file and assigned bins only in the binlister file. But I got an error saying that bins are empty!

Would be happy to get some help on this issue! Thanks.
